docs/agents-and-ingestion-tools/index.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Agents and Ingestion Tools
+nav_order: 100
+has_children: false
+has_toc: false
+---
+
+# Agents and Ingestion Tools
+
+Historically, there have been multiple popular agents and ingestion tools that work with Elasticsearch. Some of which are Beats, Logstash, Fluentd, FluentBit, and Open Telemetry. OpenSearch aims to continue to support a broad set of agents and ingestion tools, but not all have been tested or have explicitly added OpenSearch compatibility. To make getting started with OpenSearch easier, a [version value](https://github.com/opensearch-project/OpenSearch/issues/693) is being added to the OpenSearch YML. This will let you set OpenSearch 1.x clusters to report open source Elasticsearch 7.10.2. By reporting 7.10.2, the cluster will be able to connect with tools that do version checking. This is intended to be an intermediate solution while OpenSearch support is added to more tools.
```

Comment:
The version value only changes the version number that it reports as: it doesn't make OpenSearch report itself as Elasticsearch.